### PR TITLE
 added Favicon 

### PR DESCRIPTION
**Pull request by Lovegupta112**
I have added favicon in this website so that it can be attractive  and cool.

fav

### CODE REVIEW

File: contributors.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 		<title>100Devs Statistics</title>
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
```
Comment (floorrinse) [inline contributors.html:12]: Is there a reason to use the `shortcut` link type here? [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types) says it's non-conforming and suggests not to use it anymore

Comment (Lovegupta112) [reply contributors.html:12]: This method will work with all desktop browsers, old and new.but as u r saying I will check again.

Comment (floorrinse) [reply contributors.html:12]: After doing a bit more digging, I see it's used for older Internet Explorer versions, but I don't think ppl using this site would be using Internet Explorer (and using an older version of it too). I don't think we should prioritize it so I'm in favor of removing `shortcut`, but open to hear your/anyone's thoughts on this

Comment (20jasper) [reply contributors.html:12]: I don't care for supporting internet explorer personally. I'm not sure the graphs would work on it, and I don't want to spend the time to make it work for ie

Comment (20jasper) [reply contributors.html:12]: Also the favicon doesn't have a transparent background 
image

Comment (Lovegupta112) [reply contributors.html:12]: So should I change icon , i mean to say that should I change favicon

Comment (Lovegupta112) [reply contributors.html:12]: > 
@floorrinse 
U r not wrong , but for this i will need various sizes of favicon img like 16×16 ,32×32 ,ico and svg and so on.

Comment (Lovegupta112) [reply contributors.html:12]: Ok but I try another way in which we have to add .ico .png and apple touch icon ..

**Comment by Lovegupta112** [comment]
I have added favicon files by new branch enhancement

**Pull request closed without merging**